oqamaint/logger.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+#
+# implementation of a logging.Formatter to enable color output
+#
+import inspect
+import logging
+
+(BLACK, RED, GREEN, YELLOW, BLUE, MAGENTA, CYAN, WHITE) = list(range(8))
+
+RESET_SEQ = "\033[0m"
+COLOR_SEQ = "\033[1;{}m"
+
+COLORS = {
+    'WARNING': YELLOW,
+    'INFO': GREEN,
+    'DEBUG': BLUE,
+    'CRITICAL': RED,
+    'ERROR': RED}
```

Comment:
Colors are already handled using colorama, please grep for plenty of usage examples, and perhaps investigate using setup that allows for color overrides in oscrc (see `osc-staging.py`).

Reply:
no, colorama isn't in SLE ( and bot is running on a very old instance of SLE ) and we need only a few well-defined ANSI colors for better orientation in logging output

Reply:
It is in SLE 12, which was the entire reason I used it over other libraries. Perhaps an old version it is not, but that seems more like a case for updating the machine than rewriting libraries.

Reply:
unfortuanetly bot runs on LTSS versions.. and modules arent supported for ltss